src/components/navBar/navBar.js

```diff
@@ -27,21 +27,22 @@ function MdNavBar() {
     controller: MdNavBarController,
     controllerAs: 'ctrl',
     bindToController: true,
-    scope: {'selectedTabName': '=?'},
-    template: '<div class="suite md-nav-bar">' +
+    scope: {'selectedLinkName': '=?'},
+    template:
+      '<div class="md-nav-bar">' +
         '<nav role="navigation">' +
-        '<ul class="md-tab-container" layout="row" ng-transclude role="listbox"' +
-        'tabindex="0"' +
-        'ng-focus="ctrl.onFocus()"' +
-        'ng-blur="ctrl.onBlur()"' +
-        'ng-keydown="ctrl.onKeydown($event)"' +
-        'aria-label="[[Top-level navigation links|' +
-        'Description of navigation links, which provide quick access to' +
-        'main sections of the site.]]">' +
-        '</ul>' +
+          '<ul class="md-nav-bar-list" layout="row" ng-transclude role="listbox"' +
+            'tabindex="0"' +
+            'ng-focus="ctrl.onFocus()"' +
+            'ng-blur="ctrl.onBlur()"' +
+            'ng-keydown="ctrl.onKeydown($event)"' +
+            'aria-label="[[Top-level navigation links|' +
```

Comment:
I don't think there are translations like this in Material. I haven't seen them anywhere else.

@jelbourn How do aria-labels and other text content get translated? (Since as a library, you don't know what languages are supported.)